keyboards/handwired/polykybd/split72/keymaps/default/keymap.c

```diff
@@ -100,13 +100,13 @@ enum my_keycodes {
       for lang in languages:
           cog.out(f"KC_{lang}, ")
     ]]]*/
-    KC_LANG_EN, KC_LANG_DE, KC_LANG_FR, KC_LANG_ES, KC_LANG_PT, KC_LANG_IT, KC_LANG_TR, KC_LANG_KO, KC_LANG_JA, KC_LANG_AR, KC_LANG_GR,
+    KC_LANG_EN, KC_LANG_DE, KC_LANG_FR, KC_LANG_ES, KC_LANG_PT, KC_LANG_IT, KC_LANG_TR, KC_LANG_KO, KC_LANG_JA, KC_LANG_AR, KC_LANG_GR, 
```

Comment:
Seems running cog on my system led to change of whitspaces here. I use cog 3.3.0. Is there a different version recommendation?

Reply:
Yeah, not sure.. I have to check my version. Not a big deal :)